kenshin-profile-app/kenshin-profile-app/MyImage.swift

```diff
@@ -12,7 +12,7 @@ struct MyImage: View {
     var body: some View {
         Image("black")
             .resizable()
-            .frame(width: 110,height: 110)
+            .frame(width: 100,height: 100)
```

Comment:
引数と引数の間にはスペースを入れるのが一般的

```suggestion
            .frame(width: 100, height: 100)
```

他の箇所のframeもスペースないからそこも修正

Reply:
済